recipes/ogre/1.x/conanfile.py

```diff
@@ -0,0 +1,293 @@
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "cp_bin_dir": "bin",
+        "cp_media_dir": "Media",
+        "disable_plugins": False,
+        "ogre_assert_mode": 1,
+        "ogre_build_component_bites": True,
+        "ogre_build_component_hlms": True,
+        "ogre_build_component_meshlodgenerator": True,
+        "ogre_build_component_overlay": True,
+        "ogre_build_component_paging": True,
+        "ogre_build_component_property": True,
+        "ogre_build_component_python": True,
+        "ogre_build_component_rtshadersystem": True,
+        "ogre_build_component_terrain": True,
+        "ogre_build_component_volume": True,
+        "ogre_build_dependencies": False,
+        "ogre_build_plugin_bsp": True,
+        "ogre_build_plugin_octree": True,
+        "ogre_build_plugin_pcz": True,
+        "ogre_build_plugin_pfx": True,
+        "ogre_build_rendersystem_d3d11": True,
+        "ogre_build_rendersystem_gl": True,
+        "ogre_build_rendersystem_gl3plus": True,
+        "ogre_build_samples": False,
+        "ogre_build_tests": False,
+        "ogre_build_tools": True,
+        "ogre_config_enable_quad_buffer_stereo": False,
+        "ogre_config_filesystem_unicode": True,
+        "ogre_config_threads": 3,
+        "ogre_config_thread_provider": "std",
+        "ogre_config_enable_freeimage": True,
+        "ogre_install_pdb": False,
+        "ogre_install_samples": False,
+        "ogre_install_tools": True,
+        "ogre_install_vsprops": False,
+        "ogre_resourcemanager_strict": 2,
+        "ogre_static": True,
+        "ogre_set_double": False,
+        "ogre_glsupport_use_egl": True
+    }
+    exports_sources = "CMakeLists.txt", "patches/**"
+    short_paths = True
+
+    def requirements(self):
+        self.requires("cppunit/1.15.1")
+        self.requires("freeimage/3.18.0")
+        self.requires("boost/1.75.0")
+        self.requires("freetype/2.11.1")
+        self.requires("openexr/2.5.7")
+        self.requires("poco/1.11.2")
+        self.requires("tbb/2020.3")
+        self.requires("zlib/1.2.12")
+        self.requires("zziplib/0.13.71")
+        self.requires("openssl/1.1.1o", override=True)
+        self.requires("xorg/system")
+        self.requires("glu/system")
+        self.requires("sdl/2.0.20")
+        if self.options.ogre_glsupport_use_egl and self.settings.os in ["Linux", "FreeBSD"]:
+            self.requires("egl/system")
+        else:
+            self.requires("libglvnd/1.4.0")
+        
+
+    def validate(self):
+        """
+         OGRE 1.x is very old and will not work with latest gcc, clang and msvc compielrs
+         TODO: determine incompatible msvc compilers
+        """
+        if self.settings.compiler == "gcc" and tools.Version(self.settings.compiler.version) >= 9:
+            raise ConanInvalidConfiguration("OGRE 1.x not supported with gcc 11")
```

Comment:
```suggestion
        if self.settings.compiler == "gcc" and tools.Version(self.settings.compiler.version) >= 9:
            raise ConanInvalidConfiguration("OGRE 1.x not supported with gcc 11")
```
is it gcc 9 or gcc 11 ?